### PR TITLE
[IMP] accounting: payments doc update

### PR DESCRIPTION
**Pull request by toaa-odoo**
Updated the payments doc page.

Forward to master.

16.3 -> Change reconciliation section

taskid-3610834

### CODE REVIEW

**Comment by toaa-odoo** [comment]
Related PR #6651

**Review by Donapi — APPROVED** [review]
Hi Tom, 
Good job! I made a few suggestions to clarify some points, could you please have a look? :)

File: content/applications/finance/accounting/payments.rst [hunk 1]
```diff
@@ -23,69 +23,64 @@ amounts to reduce unpaid invoices/bills.
...
+     bank statement), invoices and bills are directly registered in the :guilabel:`Paid` status.
 
 Registering payments not tied to an invoice or bill
 ===================================================
 
 When a new payment is registered via the menu :menuselection:`Customers / Vendors --> Payments`, it
 is not directly linked to an invoice or bill. Instead, the account receivable or the account payable
-are matched with the outstanding account until they are manually matched with their related invoice
-or bill.
+are matched with the **outstanding account** until they are manually matched with their
```
[the remark this answers is not in the document]
Comment (toaa-odoo) [reply content/applications/finance/accounting/payments.rst:59]: IMO the concept is explained in the "outstanding" section of the :ref:, explaining it again would be redundant

File: content/applications/finance/accounting/payments.rst [hunk 2]
```diff
@@ -23,69 +23,64 @@ amounts to reduce unpaid invoices/bills.
...
      invoice.
    - If you (un)reconcile a payment in a different currency, a journal entry is automatically
      created to post the currency exchange gains/losses (reversal) amount.
    - If you (un)reconcile a payment and an invoice having cash basis taxes, a journal entry is
      automatically created to post the cash basis tax (reversal) amount.
 
-.. seealso::
-   - :doc:`bank/reconciliation`
+.. tip::
+   - If your main bank account is set as :ref:`outstanding account
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/payments.rst:50]:
```suggestion
   - If your main bank account is set as an :ref:`outstanding account
```
an/the ?

**Review by xpl-odoo — APPROVED** [review]
Pre-approved!
@robodoo delegate+

**Comment by toaa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**